libs/analytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -1068,6 +1068,9 @@ public enum Stat {
         MY_SITE_DASHBOARD_CONTEXTUAL_MENU_ACCESSED,
         PERSONALIZATION_SCREEN_CARD_HIDE_TAPPED,
         PERSONALIZATION_SCREEN_CARD_SHOW_TAPPED,
+        DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_SHOWN,
+        DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_NEW_DOMAIN_TAPPED,
+        DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_EXISTING_DOMAIN_TAPPED
```

Comment:
Thank you for adding the tracking events @justtwago 🙇 
I think those should also be added [at the end of the `AnalyticsTrackerNosara`](https://github.com/wordpress-mobile/WordPress-Android/blob/a4fa6ac15e45c3b616dc86d09529584fdd08c971/libs/analytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java#L2618):
```
            case DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_SHOWN:
                return "domain_management_use_domain_screen_shown";
            case DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_NEW_DOMAIN_TAPPED:
                return "domain_management_use_domain_screen_new_domain_tapped";
            case DOMAIN_MANAGEMENT_USE_DOMAIN_SCREEN_EXISTING_DOMAIN_TAPPED:
                return "domain_management_use_domain_screen_existing_domain_tapped";
```

Reply:
Good catch Antonis! Maybe out of scope for this task (and certainly this PR), but I wonder whether these tracker files are good candidates for conversion to Kotlin 🤔 . This isn't the first time that stat cases have been missed.

Reply:
> I wonder whether these tracker files are good candidates for conversion to Kotlin 🤔 . This isn't the first time that stat cases have been missed.

Good idea @mkevins 👍 Kotlin would help on catching the missing cases and [that file might not be that complicated to convert](https://github.com/wordpress-mobile/WordPress-Android/blob/a4fa6ac15e45c3b616dc86d09529584fdd08c971/libs/analytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java).

Reply:
Thank you! Good catch! I'd ticket the `AnalyticsTrackerNosara` conversion to Kotlin but I believe it shouldn't be placed on the `Domain Management` project's board. Where would you put it, folks?

Reply:
> Where would you put it, folks?

I don't think we have a project for this but we could add it under [the tech debt label](https://github.com/wordpress-mobile/WordPress-Android/labels/Tech%20Debt) 🤔